src/main/java/org/carlmontrobotics/commands/SwitchRPMShootNEO.java

```diff
@@ -11,22 +14,35 @@
 
 public class SwitchRPMShootNEO extends Command {
     private IntakeShooter intakeShooter;
+    private Arm arm;
+    private double currentPos;
     private double rpmAmount;
     private Timer timer = new Timer();
    
     
-    public SwitchRPMShootNEO(IntakeShooter intakeShooter) {
+    public SwitchRPMShootNEO(IntakeShooter intakeShooter, Arm arm) {
         this.intakeShooter = intakeShooter;
-        addRequirements(intakeShooter);
+        this.arm = arm;
+        addRequirements(intakeShooter, arm);
```

Comment:
if you aren't actually controlling the arm but only obtaining values from the arm, you don't need to set it as a requirement. Otherwise, you wouldn't be able to run arm commands while this is running.